ex03.ex

```diff
@@ -102,11 +109,16 @@ defmodule Ex03 do
       false
       iex> Ex03.list_equal [ 1, 2, 3 ], [3, 2, 1]
       false
+      iex> Ex03.list_equal [ 1, 2, 3, 4 ], [1, 2, 3]
+      false
 
   """
 
-  def list_equal . . . "your code"
-
+  def list_equal(h, h), do: true
```

Comment:
# -2

So... the first line is actually kind of cheating. It returns true if its two arguments are equal. Pass it two equal lists, and it returns true. This means your code could be

~~~ elixir
def list_equal(h, h), do: true
def list_equal(_, _), do: false
~~~

If you replace it with

~~~ elixir
def list_equal([], []), do: true
~~~

then your code executes the way you assume.